doc/rst/platforms/infiniband.rst

```diff
@@ -105,24 +105,27 @@ Selecting a Spawner
 Under the covers, ``gasnetrun_ibv``-based launchers must figure out
 how to spawn jobs and get them up and running on the compute nodes.
 GASNet's three ways of doing this on InfiniBand systems are ``ssh``,
-``pmi``, and ``mpi``.  When MPI is detected at configure time, it will
-be selected as the default, but we recommend using one of the other
-options if possible.
-
-* ``ssh``: Based on our experience, we recommend launching with the
-  ``ssh`` option, if possible.  This requires the ability to ``ssh``
-  to the system's compute nodes, which is not supported by all
-  systems, depending on how they are configured.  See the following
-  sub-section for details on this option.
+``pmi``, and ``mpi``, described just below.  When MPI is detected at
+configure time, it will be selected as the default, but we recommend
+using one of the other options if possible.  This can be done by
+setting the ``GASNET_IBV_SPAWNER`` environment variable, whose legal
+values are:
+
+* ``ssh``: Based on our experience, this is the preferred option, when
+  possible.  This requires the ability to ``ssh`` to the system's
+  compute nodes, which is not supported by all systems, depending on
+  how they are configured.  See the following sub-section for details
+  on this option.
```

Comment:
Might be worth a link to https://bitbucket.org/berkeleylab/gasnet/src/master/other/ssh-spawner/README as part of a "call out" that there are environment variables such as `GASNET_SSH_CMD` and `GASNET_SSH_OPTIONS` which can control how `ssh` is used; or just as "for more information".
Same for the omnipath document, of course.

Reply:
I've done this, but moved it down to the more detailed ssh-based section to try and keep these summaries briefer.  At present, only the pmi option points to the README here (because it doesn't have a more detailed section and I think doesn't currently warrant one).

> For further information about environment variables that can be used to control how `ssh` is used to launch your Chapel program, see the descriptions of ``GASNET_SSH_CMD`` and ``GASNET_SSH_OPTIONS`` in the [GASNet README for the ssh spawner](https://bitbucket.org/berkeleylab/gasnet/src/master/other/ssh-spawner/README) (also available at ``$CHPL_HOME/third-party/gasnet/gasnet-src/other/ssh-spawner/README``).